aerogear/mobile-developer-console
# Updated react-scripts

## Motivation
Updated react-scripts to allow for debugging in react tests.

## What
Updated react-scripts to allow for debugging in react tests.

## Why
Updated react-scripts to allow for debugging in react tests.

## How
Updated package.json, proxy part is no longer compatible with new `react-scripts`.
Proxy server settings script is moved to `setupProxy.js` using http-proxy-middleware. That's recommended by `react-scripts`.

## Verification Steps
1. Start with `./scripts/development.sh`
2. Check if everything works fine

## Checklist:
